examples/design/simple_reactor.py

```diff
@@ -0,0 +1,547 @@
+
+# Firstly the Plasma,
+
+# In this case `PlasmaDesigner` has some required parameters but `PlasmaBuilder` does
+# not
+
+# %%
+
+
+class PlasmaDesigner(Designer):
+    """Plasma Designer"""
+
+    param_cls = PlasmaDesignerParams
+
+    def run(self) -> GeometryParameterisation:
+        """Plasma Designer run method"""
+        return self._build_wire(self.params)
+
+    @staticmethod
+    def _build_wire(params: PlasmaDesignerParams) -> GeometryParameterisation:
+        return JohnerLCFS(
+            var_dict={
+                "r_0": {"value": params.R_0.value},
+                "z_0": {"value": params.z_0.value},
+                "a": {"value": params.R_0.value / params.A.value},
+                "kappa_u": {"value": params.kappa_u.value},
+                "kappa_l": {"value": params.kappa_l.value},
+                "delta_u": {"value": params.delta_u.value},
+                "delta_l": {"value": params.delta_l.value},
+                "phi_u_pos": {"value": params.phi_pos_u.value, "lower_bound": 0.0},
+                "phi_u_neg": {"value": params.phi_neg_u.value, "lower_bound": 0.0},
+                "phi_l_pos": {"value": params.phi_pos_l.value, "lower_bound": 0.0},
+                "phi_l_neg": {
+                    "value": params.phi_neg_l.value,
+                    "lower_bound": 0.0,
+                    "upper_bound": 90,
+                },
+            }
+        )
+
+
+class PlasmaBuilder(Builder):
+    """
+    Our PlasmaBuilder
+    """
+
+    param_cls = None
+
+    def __init__(self, wire: BluemiraWire, build_config: Dict):
+        super().__init__(None, build_config)
+        self.wire = wire
+
+    def build(self) -> Plasma:
+        """
+        Run the full build of the Plasma
+        """
+        xz = self.build_xz()
+        return Plasma(
+            self.component_tree(
+                xz=[xz],
+                xy=[Component("")],
+                xyz=[self.build_xyz(xz.shape)],
+            )
+        )
+
+    def build_xz(self) -> PhysicalComponent:
+        """
+        Build the xz Component of the Plasma
+        """
+        component = PhysicalComponent("LCFS", BluemiraFace(self.wire))
+        component.display_cad_options.color = BLUE_PALETTE["PL"]
+        component.display_cad_options.transparency = 0.5
+        return component
+
+    def build_xyz(self, lcfs: BluemiraFace) -> PhysicalComponent:
+        """
+        Build the xyz Component of the Plasma
+        """
+        shape = revolve_shape(lcfs, degree=359)
+        component = PhysicalComponent("LCFS", shape)
+        component.display_cad_options.color = BLUE_PALETTE["PL"]
+        component.display_cad_options.transparency = 0.5
+        return component
+
+
+# %%[markdown]
+
+# And now the TF Coil, in this instance for simplicity we are only making one TF coil.
+# If more TF coils were to be required the build_xyz of `TFCoilBuilder` would need to
+# be modified.
```

Comment:
We should give a quick run-through of exactly what this designer is doing. It's finding the given geometry parameterisation, then feeding that into the optimisation problem we made earlier, then running the optimisation problem to generate the centreline wire